### DISCUSSION TITLE
performance

### DISCUSSION THREAD

**Discussion by Forgowski** [fedorovvl/tso_client#303] [Q&A]
Hi is there any option to change the performance of the client? I thought about RAM and FPS limit is there any option to change it?

**Comment by fedorovvl** [comment]
hi. 
ram not limited, x86 can use up to 3gb ram (arch specific), x64 has no limit
fps hardcoded (mb i can change it, but what for?)

**Reply by Forgowski** [reply]
through the game site, the game is smoother. I was wondering if there is a possibility to create the same impression in the client :)
